### PR TITLE
Clean up swig bindings

### PR DESCRIPTION
**Pull request by btovar**
Only functions in taskvine.h are used in the python bindings.

## Merge Checklist

The following items must be completed before PRs can be merge.
Check these off to verify you have completed all steps.

- [x] `make test`       Run local tests prior to pushing.
- [x] `make format`     Format source code to comply with lint policies. Note that some lint errors can only be resolved manually (e.g., Python)
- [x] `make lint`       Run lint on source code prior to pushing.
- [ ] Manual Update     Update the manual to reflect user-visible changes.
- [x] Type Labels       Select a github label for the type: bugfix, enhancement, etc.
- [x] Product Labels    Select a github label for the product: TaskVine, Makeflow, etc.
- [x] PR RTM            Mark your PR as ready to merge.

### CODE REVIEW

**Review by dthain — APPROVED** [review]

**Review by JinZhou5042 — APPROVED** [review]

File: taskvine/src/bindings/python3/ndcctools/taskvine/manager.py [hunk 1]
```diff
@@ -245,13 +245,13 @@ def staging_directory(self):
...
             return self._staging_explicit
         else:
-            return cvine.vine_get_runtime_path_staging(self._taskvine, None)
+            return cvine.vine_get_path_staging(self._taskvine, None)
 
     ##
     # Get the caching directory of the manager
     @property
     def cache_directory(self):
-        return cvine.vine_get_runtime_path_caching(self._taskvine, None)
```
Comment (tphung3) [inline taskvine/src/bindings/python3/ndcctools/taskvine/manager.py:254]: Is there a reason we use the staging directory instead of the cache directory? I probably missed out on why this change was here..
